gitlab/assets/dashboards/overview.json

```diff
@@ -0,0 +1,419 @@
+{
+    "author_info": {
+        "author_name": "Datadog"
+    },
+    "board_title": "Gitlab Overview",
+    "created": "2020-03-13T15:43:50.298533+00:00",
+    "created_by": {
+        "access_role": "st",
+        "disabled": false,
+        "email": "support@datadoghq.com",
+        "handle": "support@datadoghq.com",
+        "is_admin": false,
+        "name": "Datadog",
+        "role": null,
+        "title": null,
+        "verified": true
+    },
+    "description": "This dashboard provides a high-level overview of your Gitlab instance so you can monitor metrics related to pipelines, repositories, requests, and more.\n\nFor further reading on monitoring Gitlab, see: [Official Gitlab integration docs](https://docs.datadoghq.com/integrations/gitlab/)\n\nClone this template to make changes and add your own graphs and widgets.",
+    "id": 1014699,
+    "modified": "2020-03-30T18:03:40.950891+00:00",
+    "new_id": "c9w-dnm-m3a",
+    "read_only": false,
+    "template_variables": [
+        {
+            "default": "*",
+            "name": "gitlab_host",
+            "prefix": "gitlab_host"
+        },
+        {
+            "default": "*",
+            "name": "gitlab_port",
+            "prefix": "gitlab_port"
+        }
+    ],
+    "widgets": [
+        {
+            "height": 9,
+            "id": 0,
+            "sizing": "zoom",
+            "type": "image",
+            "url": "https://static.datadoghq.com/static/images/saas_logos/bot/gitlab@2x.png",
+            "width": 15,
+            "x": 1,
+            "y": 1
+        },
+        {
+            "check": "gitlab.liveness",
+            "group": "gitlab_host:localhost,gitlab_port:8086,optional:tag1,host:docker-desktop",
+            "group_by": [],
+            "grouping": "cluster",
+            "height": 9,
+            "id": 1,
+            "tags": [
+                "*"
+            ],
+            "title": true,
+            "title_align": "center",
+            "title_size": "16",
+            "title_text": "Liveness",
+            "type": "check_status",
+            "width": 14,
+            "x": 18,
+            "y": 1
+        },
+        {
+            "check": "gitlab.readiness",
+            "group_by": [],
+            "grouping": "cluster",
+            "height": 8,
+            "id": 2,
+            "tags": [
+                "*"
```

Comment:
Ditto, let's add the template variables here, so that when users update them the value of this widget reflects that information.

Reply:
Added!